channels/applicationinsights-channel-js/src/Sender.ts

```diff
@@ -325,41 +344,48 @@ export class Sender extends BaseTelemetryPlugin implements IChannelControls {
                     } else {
                         _customHeaders = null;
                     }
-
                     _enableSendPromise = senderConfig.enableSendPromise;
 
-                    let sendPostFunc: SenderFunction = null;
-                    if (!senderConfig.disableXhr && useXDomainRequest()) {
-                        sendPostFunc = _xdrSender; // IE 8 and 9
-                    } else if (!senderConfig.disableXhr && isXhrSupported()) {
-                        sendPostFunc = _xhrSender;
-                    }
-    
-                    if (!sendPostFunc && isFetchSupported()) {
-                        sendPostFunc = _fetchSender;
-                    }
+                    let customInterface = senderConfig.httpXHROverride;
+                    let httpInterface: IXHROverride = null;
+                    let fallbackInterface: IXHROverride = null;
+                    let syncInterface: IXHROverride = null;
+                    httpInterface = _getSenderInterface([TransportType.Xhr, TransportType.Fetch], false);
+                  
+                    // *****************************************************************************************************************
+                    //NOTE: shoud we remove fallback sender?
```

Comment:
👍

Reply:
removed `fallbackSender` and use `xhrSender` directly.

**Previous**: 
when using `beaconSender`, if the payload is too large, it will try

- fetch(when disableXhr is true && fetch is supported)
- xhr (other scenarios)

**Current:**

- if payload is too large, use xhr direcltly.

**1DS:**

-  will not use fallback sender, will only send notification.

(so maybe, we should keep `fallbackSender`?)